superdwarf-lua.lua

```diff
@@ -0,0 +1,126 @@
+-- makes units very speedy
+--@ module = true
+local help = [====[
+
+superdwarf
+==============
+Similar to `fastdwarf`, per-creature.
+Overrides units timers every game tick to 1, causing extremely fast actions
+
+Usage:
+
+Valid Options:
+
+``-unit <id>``: Targets the unit with the specified ID.
+                This is optional; if not specified, the selected unit is used instead.
+                
+
+``-help``:      Shows this help information
+
+
+]====]
+
+local repeatUtil = require('repeat-util')
+local utils = require('utils')
+local validArgs = utils.invert({
+    'add',
+    'del',
+    'clear',
+    'unit',
+    'list',
+    'help',
+})
```

Comment:
This changes the script to take arguments like `-add`, `-list`, etc. instead of the old `add`, `list`. It would be nice if we could keep compatibility. Also, this allows any number of these arguments to be specified, which is probably not necessary - the old version only allowed one at a time.

You can extract the first argument with something like:

```lua
local commandArg = ({...})[1]
```

Reply:
Sure, didn't realize compatibility was a fairly large ocncern